### PR TITLE
4.1 - v4 Migration Cleanup

### PR DESCRIPTION
**Pull request by rejahrehim**
This PR covers issue #<issue number>.

- [x] This PR handles the issue and requires no additional PRs.
- [x] You have validated the need for this change.

**What did this PR accomplish?**

- Fix linting issue

### CODE REVIEW

File: document/4_Web_Application_Security_Testing/4.1_Introduction_and_Objectives/4.1_Testing_Introduction_and_Objectives.md [hunk 1]
```diff
@@ -28,7 +34,8 @@ This approach tends to create a defined Testing Methodology that will be:
 
 The problems to be addressed are fully documented and tested. It is important to use a method to test all known vulnerabilities and document all the security test activities.
 
-### What is the OWASP testing methodology?
+## What is the OWASP testing methodology?
```
Comment (kingthorin) [inline document/4_Web_Application_Security_Testing/4.1_Introduction_and_Objectives/4.1_Testing_Introduction_and_Objectives.md]: Title caps please

File: document/4_Web_Application_Security_Testing/4.1_Introduction_and_Objectives/4.1_Testing_Introduction_and_Objectives.md [hunk 2]
```diff
@@ -39,7 +46,7 @@ The testing model consists of:
 
 The test is divided into 2 phases:
 
-#### Phase 1 Passive mode:
+### Phase 1 Passive mode:
```
Comment (kingthorin) [inline document/4_Web_Application_Security_Testing/4.1_Introduction_and_Objectives/4.1_Testing_Introduction_and_Objectives.md]: Mode: too

File: document/4_Web_Application_Security_Testing/4.1_Introduction_and_Objectives/4.1_Testing_Introduction_and_Objectives.md [hunk 3]
```diff
@@ -55,7 +62,7 @@ The following parameters represent two access points (gates) to the application:
 
 In this case, the application shows two gates (parameters a and b). All the gates found in this phase represent a point of testing. A spreadsheet with the directory tree of the application and all the access points would be useful for the second phase.
 
-#### Phase 2 Active mode:
+### Phase 2 Active mode:
```
Comment (kingthorin) [inline document/4_Web_Application_Security_Testing/4.1_Introduction_and_Objectives/4.1_Testing_Introduction_and_Objectives.md]: Mode: here too

Comment (rejahrehim) [reply document/4_Web_Application_Security_Testing/4.1_Introduction_and_Objectives/4.1_Testing_Introduction_and_Objectives.md]: Done

**Pull request merged**